conf/local.conf.sample

```diff
@@ -48,275 +46,83 @@ MACHINE ??= "qemux86"
 # There are other options that can be used here, see
 # meta/classes/image.bbclass and meta/classes/core-image.bbclass for more
 # details.
+EXTRA_IMAGE_FEATURES ?= "multilib-runtime"
 
+# Image features for development-image
 IMAGE_FEATURES_DEVELOPMENT ?= "debug-tweaks"
 IMAGE_FEATURES_DEVELOPMENT:append:feature-flex-bsp = " codebench-debug ssh-server-openssh tools-profile"
-EXTRA_IMAGE_FEATURES = "${IMAGE_FEATURES_DEVELOPMENT} multilib-runtime"
 
 # Image features for production-image
 IMAGE_FEATURES_PRODUCTION ?= ""
 IMAGE_FEATURES_DISABLED_PRODUCTION ?= "${IMAGE_FEATURES_DEVELOPMENT} ssh-server-dropbear"
 
-# Install tzdata with systemd when building read only images
-CORE_IMAGE_EXTRA_INSTALL += "${@bb.utils.contains('IMAGE_FEATURES', 'read-only-rootfs', 'tzdata' if d.getVar('VIRTUAL-RUNTIME_init_manager') == 'systemd' else '', '', d)}"
-
-# Uncomment to enable runtime testing with ptest
-#USER_FEATURES += "ptest"
-#EXTRA_IMAGE_FEATURES += "ptest-pkgs"
-
-# Example of how to add additional image types beyond the default. If the
-# machine doesn't define them, the default types are tar.bz2 and ext3.
-#IMAGE_FSTYPES:append = " jffs2"
-
-# Uncomment and alter to set the root password. Set to '0' to explicitly allow
-# the empty root password even in production-image. Set to '*' to explicitly
-# disable root login.
+# Set the root password. Set to '0' to explicitly set and allow the empty root
+# password even in production-image. Set to '*' to explicitly disable root
+# login.
 #ROOT_PASSWORD = "0"
 IMAGE_CLASSES:append = " image-root-password"
 
-# User features lets you manipulate the distro features. To add a distro
-# feature, simply add it to USER_FEATURES. To remove, prefix it with ~.
-#
-# # Disable x11
-# USER_FEATURES += "~x11"
-#
-# Disable 3g by default, as most won't need it.
-USER_FEATURES += "~3g"
+# Enable and disable distro features. To disable a feature, prefix it with ~
+#USER_FEATURES += "~x11"
```

Comment:
In my opinion we are not using it so we should remove it from here

Reply:
USER_FEATURES is used everywhere, it's our primary way of enabling and disabling features. Admittedly we have other examples of it when local.conf fragments are included for such features, but it's both commonly used for our product and is specific to us, not in the Yocto docs. I'm unsure, I think this needs discussion.